packages/plugin-ext/src/main/browser/webviews-main.ts

```diff
@@ -107,15 +164,30 @@ export class WebviewsMainImpl implements WebviewsMain {
         }
     }
     $reveal(handle: string, showOptions: WebviewPanelShowOptions): void {
-        const webview = this.getWebview(handle);
-        if (webview.isDisposed) {
+        const view = this.getWebview(handle);
+        if (view.isDisposed) {
             return;
         }
-        // FIXME handle view column here too!
+        if (showOptions.viewColumn !== undefined || (showOptions.area !== undefined && showOptions.area !== 'main')) {
```

Comment:
`showOptions.area !== undefined && showOptions.area !== 'main'` 

One can move from side panel to main, but not another way around? Did you mean `this.shell.getAreaFor(view) !== showOptions.area`?

Reply:
fixed